LBHackney-IT/lbh-fss-portal-frontend
# Feature/export active users

**Description**

Add functionality to download a csv of all active users 

**Test instructions**

- pull this branch 
- `yarn install`
- navigate to **/users**
- click 'Export users' to download csv of active users
- the following fields should be contained in the csv 
    - ID
    - Name
    - Email
    - Organisation
    - VCSO user
    - Viewer user
    - Admin user
    - Status (filtered to only include active users)

**Note 1** - the ticket does not specify a list of fields to include, I have therefore judged these to be the most relevant fields 

**Note 2** - there is no figma designs of the button, I have therefore used the same styling as other secondary buttons on the site 

**Ticket**

https://app.clickup.com/t/9e5u96
